Makefile

```diff
@@ -91,12 +87,15 @@ $(BUILD_DIR)/linux-amd64/crc: $(SOURCES)
 $(BUILD_DIR)/windows-amd64/crc.exe: $(SOURCES)
 	GOARCH=amd64 GOOS=windows go build -ldflags="$(LDFLAGS)" -o $(BUILD_DIR)/windows-amd64/crc.exe ./cmd/crc
 
+build-aux/crc-embedder: $(SOURCES) build-aux/crc-embedder.go
```

Comment:
why we require `build-aux/crc-embedder.go` for this target? I tested it on mac without adding it and it does work without any issue.

Reply:
Ah right it should be part of $(SOURCES). I'll try to drop it.